runtime/onert/core/src/compiler/train/TrainingCompiler.cc

```diff
@@ -194,8 +208,8 @@ std::shared_ptr<CompilerArtifact> TrainingCompiler::compile(void)
   for (const auto &pair : lowered_subgs)
   {
     const auto &subg_index = pair.first;
-    const auto &lowered_subg = pair.second;
-    dot_dumper.dump(*lowered_subg, nnfw::misc::str("after_lower_subg-", subg_index.value()));
+    const auto &subg = pair.second;
+    dot_dumper.dump(*subg, nnfw::misc::str("after_lower_subg-", subg_index.value()));
```

Comment:
`pair.second` is not a `TrainableGraph`. If you want to call `DotDumper::dump(const ir::train::TrainableGraph &graph, const std::string &tag)`, how about changing this code like below?
```suggestion
    const auto &trainable_subg = pair.second->trainable_graph();
    dot_dumper.dump(trainable_subg, nnfw::misc::str("after_lower_subg-", subg_index.value()));
```

Reply:
I think it is beyond this PR. I just kept the original working. Do you want to fix this in this PR?

Reply:
@ragmani I removed the change for dumping lowered graph.


I don't understand fully about original code. It is the reason I kept the original behavior. It uses `ILoweredGraph` version.

```cc
void DotDumper::dump(const compiler::ILoweredGraph &lowered_graph, const std::string &tag)
```

not
```cc
void DotDumper::dump(const ir::Graph &graph, const std::string &tag)
```

It is beyond this PR.